docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -0,0 +1,53 @@
+.. _2017_risk_exposure_template:
+
+----------------------
+Risk Exposure Template
+----------------------
+
+.. todo::
+  Add instructions, following Nathaniel's cause model template instructions. Flesh out all headings.
+
+
+Risk Exposure Overview
+----------------------
+
+ - expert opinion
+
+Risk Exposure description
+-------------------------
+
+Risk Exposures Description in GBD
+------------------------------
+
+  - Type of risk (categorical, continuous, etc.)
+
+  - What is this a risk for?
+
+  - TMREL (high level overview. does it vary by outcome?)
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Cause Model Diagram

```

Comment:
Is this a 'Cause Model' diagram or 'Risk Exposure' diagram? Or would it include 'Risk Exposure' & 'Causes' it impacts?

Reply:
risk exposure! thanks for the catch

Reply:
Right, there shouldn't be a cause model diagram. I don't think there's an analogue for risks, but yes, we should think of other things we need for risk exposures.